src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1402,18 +1402,16 @@ InferenceProfiler::SummarizeSendRequestRate(
   summary.send_request_rate = num_sent_requests / window_duration_s;
 }
 
-cb::Error
-InferenceProfiler::SummarizeServerStatsHelper(
+int64_t
```

Comment:
(nit) Reorder this under the function that uses it?

Reply:
done